modules/microshift-kubeconfig-hosts-ips-config.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_networking/microshift-kubeconfig.adoc
+
+:_content-type: PROCEDURE
+[id="Configuring-hosts-ips_{context}"]
+= Configuring host names and IP addresses
+
+You can configure additional host names and IP addresses on top of the defaults, which are `<node.hostnameOverride>`, `<node.nodeIP>` and `api.<dns.baseDomain>` using the remote access `kubeconfig` file.
+
+[NOTE]
+====
+You must restart {product-title} for configuration changes to be implemented.
+====
+
+.Prerequisites
+
+* You have installed the `oc` CLI tool.
+//Q: Do we actually need this or kubectl or no?
+* You have created a `config.yaml` for {product-title}.
+//Q: Need to do this, or no?
+//Q: Does configuration here take precedence over a config.yaml if one was created? Or can it otherwise create conflict we should note?
+
+.Procedure
+
+. To show the contents of the existing remote-access `kubeconfig` file, run the following command:
++
+[source, terminal]
+----
+$ cat /var/lib/microshift/resources/kubeadmin/kubeconfig
```

Comment:
Keep in mind that this path is the local kubeconfig, the remote ones follow a different path:
`/var/lib/microshift/resources/kubeadmin/<name/IP address>/kubeconfig`

The default remote access is the one under `/var/lib/microshift/resources/kubeadmin/$HOSTNAME/kubeconfig` which is always generated.